packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -301,7 +302,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           onDismiss={ this._onDismiss }
           onPositioned={ this._onPositioned }
         >
-          <div style={ { width: this._dropDown.clientWidth - 2 } }>
+          <div style={ { width: dropdownWidth || this._dropDown.clientWidth - 2 } }>
```

Comment:
Issue is width is being set on the wrong container 
![image](https://user-images.githubusercontent.com/1434956/28989246-f5f1b276-7928-11e7-8b67-30111c6afac8.png)